dart/lib/async/html.dart

```diff
@@ -278,7 +278,11 @@ class _ElementPageLoaderElement extends HtmlPageLoaderElement {
   Future<Rectangle> get offset async => node.offset;
 
   @override
-  Future click({bool sync: true}) => loader.executeSynced(() {
+  Future click({bool sync: true}) => loader.executeSynced(() async {
+        await _microtask(() =>
```

Comment:
I think these should be nested inside the else branch below (not sure they will work as expected for option elements etc.)